check-headers.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+files=$(find src -name \*.js -type f -print0 | xargs -0 egrep -L '(Licensed under the Apache License)|(Code generated from|by)')
+if [ -n "$files" ]; then
+  echo "Missing license header in:"
+  echo "$files"
+  exit 1
+fi
```

Comment:
How is this part working? 😅
I'm guessing that we first try to find file with missing line(s) and then if we do - we output them and exit with 1?
And if there are no such filex we exit with 0? 🤔

Reply:
> we first try to find file with missing line(s) and then if we do - we output them and exit with 1?
And if there are no such filex we exit with 0?

Exactly! And CI will abort for exit code != 0